setup.cfg

```
@@ -59,7 +60,7 @@ include =
 spectacular =
     drf-spectacular
 tests =
-    drf-spectacular
+    drf-spectacular>=0.26.4
```

```suggestion
    drf-spectacular
```
een minimale versie is volgens mij niet nodig, CI zal altijd de meest recente versie installeren